vizstorm_sites/vizstorm_app/server.R

```diff
@@ -72,22 +94,30 @@ shiny::shinyServer(function(input, output,session) {
     } else {
       siteDF[["picked_sites"]] <- unique(c(siteDF[["picked_sites"]], clicked_site$id))
     }
-
-    proxy %>% selectRows(which(siteDF[["lat_lon"]]$site_no %in% siteDF[["picked_sites"]]))
-    
+    siteDF[["lat_lon"]][["picked_sites"]] <- siteDF[["lat_lon"]][["site_no"]] %in% siteDF[["picked_sites"]]
+    proxy %>% selectRows(which(siteDF[["lat_lon"]][["picked_sites"]]))
+      
   })
   
   plot_sparks <- eventReactive(input$showSparks,{
+    # Next step....
+    # Turn this facetted ggplot2 into:
+    # https://leonawicz.github.io/HtmlWidgetExamples/ex_dt_sparkline.html
+    # Then...Let that table *also* click on/off sites.
+    
     x <- siteDF[["stream_data"]]
     
     sites_to_show <- siteDF[["picked_sites"]]
     
     x <- filter(x, site_no %in% sites_to_show)
+
+    x <- left_join(x, select(siteDF[["lat_lon"]], station_nm, site_no), by="site_no")
+    x$name_num <- paste(x$station_nm, x$site_no, sep = "\n")
     
     sparklines <- ggplot(data = x) + 
       geom_line(aes(x=dateTime, y=X_00065_00000),size = 1) +
       geom_line(data = filter(x, flooded), aes(x=dateTime, y=X_00065_00000),size = 3, color = "blue") +
```

Comment:
maybe make this line into a geom_point so that the dots just enhance the existing line rather than drawing their own line (as in 2nd site below)?
![image](https://user-images.githubusercontent.com/12039957/46751813-a07e0780-cc89-11e8-9528-40cd19dc2597.png)